src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      { age
+        && (
+        <p className="Person__age">
+          {'I am '}
+          {age}
+        </p>
+        )}
+      <p className="Person__partner">
+        { isMarried
+          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
+          : 'I am not married'}
+      </p>
```

Comment:
```suggestion
      <h2 className="Person__name">
        {'My name is '}
        {name}
      </h2>
      
      { age
        && (
        <p className="Person__age">
          {'I am '}
          {age}
        </p>
        )}
        
      <p className="Person__partner">
        { isMarried
          ? `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
          : 'I am not married'}
      </p>
```

Reply:
I think you should add spaces between blocks to improve code readability.

Reply:
Agree about spaces

Reply:
fixed all